### DISCUSSION TITLE
Maintain tab context of repo when moving through project view

### DISCUSSION THREAD

**Discussion by roblevintennis** [community/community#9029] [Projects and Issues]
I wish I didn't lose the tab context when moving into the beta projects. For example, if in Issues, or Discussions, etc., I can tab between these sections of my repo. When I'm in the new beta projects I feel "stuck"

**Comment by Mikibits** [comment]
I feel exactly the same way. The lack of navigation further adds to the confusion where there are top-level projects and repo-level projects and it's not clear what the difference is, nor is there a way to navigate between them. (This is probably worth a separate thread).

**Comment by MichaelTiemannOSC** [comment]
Suggest changing title to "Maintain tab context of repo when moving through project view" (or whatever is the best wording for what you are trying to do).

**Reply by roblevintennis** [reply]
Agreed @MichaelTiemannOSC I think I just used whatever tooling they had in place and didn't think to change title. Thanks for recommendation. I used yours as it makes perfect sense 🍰 💯

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
